src/components/combo/types.ts

```diff
@@ -35,10 +36,10 @@ export interface GroupingOptions<T extends object> {
 }
 
 /* marshalByValue */
-export interface IgcComboChangeEventArgs {
-  newValue: string[];
+export interface IgcComboChangeEventArgs<T extends object = any> {
+  newValue: ComboValue<T>;
```

Comment:
Figured why the analyzer turned this into object. Um, this should still be array tho, right?
```suggestion
  newValue: ComboValue<T>[];
```
Otherwise if you type the event args proper (Not a bad idea to begin with):
```ts
it('should fire igcChange selection type event on mouse click', async () => {
      const eventSpy = sinon.spy(combo, 'emitEvent');
      const args: Partial<CustomEvent<IgcComboChangeEventArgs<City>>> = {
        cancelable: true,
        detail: {
          newValue: ['BG02'],
          items: [cities[1]],
          type: 'selection',
        },
      };
```
It's really not happy :)
![image](https://github.com/IgniteUI/igniteui-webcomponents/assets/3198469/fda52806-c77a-431c-ba18-72a7b9066e75)

Reply:
Yup, just noticed, same as `value` already is.